exercises/Functions-6-R.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: exercise
+title: Functions 6
+language: R
+---
+
+This is a follow up to [Strings 6]({{ site.baseurl }}/exercises/Strings-6/).
+
+A colleague has produced a file with one DNA sequence on each line. Download
+[the file]({{ site.baseurl }}/data/dna_sequences_1.txt) and load it into R using
+`read.csv()`. The file has no header and is separated by white space (`’’`).
+
+Calculate the GC content\* of each sequence by writing a function that takes a
+dna sequence as input and returns the GC-content of that sequence. Print the
+result for each sequence. Before we knew about functions we had to take each dna
+sequence one at a time and then rewrite or copy-paste the same code to analyze
+each one. Isn't this better?
+
+\* The GC content is the percentage of bases that are either G or C (as a
+percentage of total base pairs).
```

Comment:
Instead of the * what about just add the description immediately following the original mention:

"Calculate the GC content, the percentage of bases that are either G or C as a percentage of total base pairs,  of each sequence. Do this by writing a function that takes a dna sequence as input and returns the GC-content of that sequence."